core/dpdk.cc

```diff
@@ -152,7 +152,38 @@ static void init_eal(const char *prog_name, int mb_per_socket,
   rte_openlog_stream(fopencookie(nullptr, "w", dpdk_log_funcs));
 }
 
+static int determine_default_core() {
```

Comment:
Could you add a comment saying that this returns the last core that is available, which is the "default" for use by non-workers?